seedDB/README.md

```diff
@@ -1,17 +1,20 @@
-These files are MySQL database dump for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with:
+# cBioPortal Seed Database
 
-- DB schema version **1.3.0**
+These files are MySQL database dump files for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with:
+
+- DB schema version **2.1.0**
 
 Which is suitable for:
 
-- release version **1.3.1** of cBioPortal.  
-- release version **> 1.3.1** of cBioPortal (:warning: with possible migration script step). 
+- release version **1.5.0** of cBioPortal.  
+- release version **> 1.5.0** of cBioPortal (:warning: with possible migration script step). 
 
 :information_source: the DB schema and the cBioPortal can follow different numbering cycles. This means that the version numbers won't be always identical. 
 
 You can download the files by using the links below:
 
-- **Schema 1.3.0**: [SQL file with create table statements for portal release 1.3.1](https://raw.githubusercontent.com/cBioPortal/cbioportal/v1.3.1/core/src/main/resources/db/cgds.sql) 
-- **Seed data, part1**: [cbioportal-seed SQL (.gz) file - part1 (no pdb_ tables)](https://github.com/cbioportal/datahub/raw/b69c86803c40d543080bf31a645721d06c82d08d/seedDB/seed-cbioportal_no-pdb_hg19.sql.gz)
-- **Seed data, part2 (optional)** [cbioportal-seed SQL (.gz) file - part2 (only pdb_ tables)](https://github.com/cbioportal/datahub/raw/b69c86803c40d543080bf31a645721d06c82d08d/seedDB/seed-cbioportal_only-pdb.sql.gz)
+- **Schema 2.1.0**: [SQL file with create table statements for portal release 1.5.0](https://raw.githubusercontent.com/cBioPortal/cbioportal/v1.5.0/db-scripts/src/main/resources/cgds.sql) 
+- **Seed data, part1**: [cbioportal-seed SQL (.gz) file - part1 (no pdb_ tables)](seed-cbioportal_hg19_v2.1.0.sql.gz)
+- **Seed data, part2 (optional)** [cbioportal-seed SQL (.gz) file - part2 (only pdb_ tables)](seed-cbioportal_hg19_v2.1.0_only-pdb.sql.gz)
```

Comment:
by introducing version in the file name, other parts of documentation also need to change:
- the docker documentation in https://github.com/thehyve/cbioportal-docker/blob/master/README.md. Please make a PR there to reflect the fact that version nr part of the file name now.
- make this version nr more explicit in your new doc `Update-Seed-Database.md`